test/promql/tests/data_plane/gitops_sync_test.yaml

```diff
@@ -0,0 +1,86 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
```

Comment:
In reality, this metric also has labels for identifying the apps for which the info is provided (namespace, name, etc.). We need test cases with some of those labels to see that the computations work as expected.

Reply:
I added the namespace="gitops-service-argocd" to each expression

Reply:
but they are all the same namespace. In reality, you have time series for multiple apps. Your current tests don't test the way you handle multiple apps.

Reply:
Moreover, the SLO itself, IIUC, is about summarizing multiple apps, so I can't see it serves purpose without providing additional time series for additional apps.

Reply:
From the GitOps service POV, I think all our apps are deployed to the same namespace. I can double check with Jonathan.

Reply:
I added multiple apps/series.  We can add more if needed.